### ISSUE TITLE
[NPC] [Quest] Skywing

### ISSUE THREAD

**Issue by bekonja** [Atlantiss/NetherwingBugtracker#4121]
[//]: # (Enclose links to things related to the bug using http://wowhead.com or any other TBC database.)
[//]: # (You can use screenshot ingame to visual the issue.)
[//]: # (Write your tickets according to the format:)
[//]: # ([Quest][Azuremyst Isle] Red Snapper - Very Tasty!)
[//]: # ([NPC] Magistrix Erona)
[//]: # ([Spell][Mage] Fireball)
[//]: # ([Npc][Drop] Ghostclaw Lynx)
[//]: # ([Web] Armory doesnt work)
[//]: # (To find server revision type `.server info` command in the game chat.)

**Description**: Skywing mob is ingame but gives me no escort quest. Smaily told me that it may be the case of quest missing due it being added in later patch (2.1).

**Current behaviour**:
Skywing mob is ingame but gives me no escort quest.
https://db.excalibur.ws/?npc=22424
https://db.excalibur.ws/?quest=10898

**Expected behaviour**: Either remove the mob or add the quest.

**Server Revision**: 

EDIT1: 
https://www.wowhead.com/quest=10898/skywing#comments
Earliest comments are from 2.0.5 patch, quest should work on launch.

EDIT2:
My guildies reported this quest working just yesterday. Its an issue of mob glitching and not giving it for reason unknown.

EDIT3: Managed to reset it by myself by pulling mobs into Skywing and killing it. When it respawned quest started working. Reason behind it not working in a first place is unknown.

**Comment by wtRCH** [comment]
@aresbrutus

**Comment by Evolvee** [comment]
cant reproduce

**Comment by Justfor49** [comment]
This bug is still happening. Went to Skywing and he had no quest to gave me (no yellow exclamation mark, couldn't speak with him). So I did what the user said above and got nearby mobs to aggro on him and kill him. After Skywing was killed he respawned quickly and then he had the yellow exclamation mark above his head and I was able to accept quest like normal. See images below:

NO QUEST AVAILABLE:
![skywing 1](https://user-images.githubusercontent.com/69169443/90816646-a075d680-e2fa-11ea-843b-80c821191df8.jpg)

QUEST AVAILABLE AFTER SKYWING DIES AND RESPAWNS (notice his dead body near him):
![skywing 2](https://user-images.githubusercontent.com/69169443/90816680-b4b9d380-e2fa-11ea-85f4-e2b1d6e2dbbf.jpg)
